language/move-lang/tests/functional/sliding_nonce/test.move

```diff
@@ -49,3 +50,29 @@ script {
     }
 }
 // check: EXECUTED
+
+//! new-transaction
+//! sender: vivian
+script {
+    use 0x0::SlidingNonce;
+
+    fun main() {
+        SlidingNonce::publish_nonce_resource_for_user();
+        SlidingNonce::record_nonce_or_abort(1);
+        SlidingNonce::record_nonce_or_abort(1);
+    }
+}
+// check: ABORTED
+// check: 10003
+
+//! new-transaction
+//! sender: vivian
+script {
+    use 0x0::SlidingNonce;
+
+    fun main() {
+        SlidingNonce::publish_nonce_resource_for_user();
+        SlidingNonce::record_nonce_or_abort(128);
```

Comment:
@andll Are there additional checks that should be done here with a nonce above 128?

Reply:
Mm, I am not sure I understand context. Are those tests different from existing tests for bob?

Reply:
This follows a different code path in the sliding nonce module is all (since this is different for nonce >= 128), so I was just checking to see if this was any different from the `bob` checks is all.

Reply:
Got it. Thanks for adding extra test case, and glad that coverage caught that!

I think most of other >128 test cases are covered in bob tests